crates/cdk/src/nuts/nut02.rs

```diff
@@ -490,10 +509,28 @@ mod test {
     fn test_to_int() {
         let id = Id::from_str("009a1f293253e41e").unwrap();
 
-        let id_int = u64::try_from(id).unwrap();
+        let id_int = u32::try_from(id).unwrap();
         assert_eq!(864559728, id_int)
     }
 
+    #[test]
+    fn test_u64_to_id_and_back_conversion() {
```

Comment:
This function name is innaccurate and too long. Change it to something like this:
```suggestion
    fn test_to_u64_and_back() {
```